exercise-1.js

```diff
@@ -228,6 +350,7 @@
                 console.log("welcome to the 21st century");
```

Comment:
The one refactoring which was attempted in this section doesn't quite meet the requirements.  The other two weren't attempted.